File: test1/src/ServerClient/Follower.java
package ServerClient;

public class Follower {
	private String id;
	private Client client;
	
	public Follower(String id,Client client){
		this.id=id;
		this.client=client;
	}
	public String getID(){
		return this.id;
	}
	public Client getClient(){
		return this.client;
	}

}

File: clientSide/src2/Follower.cpp
/*
 * Follower.cpp
 *
 *  Created on: Jan 13, 2014
 *      Author: ofri
 */

#include "../include2/Follower.h"
#include <string>
#include <iostream>
#include <vector>


using namespace std;


Follower::Follower(string userName,int id){
	this->usersName=userName;
	this->id=id;
}
int Follower::getID(){
	return this->id;
}
string Follower::getUserName(){
	return this->usersName;
}
Follower::~Follower(){

}
